azuredeploy.json

```diff
@@ -567,6 +567,13 @@
                 "description": "Virtual network gateway vpn type"
             },
             "type": "string"
+        },
+        "loadBalancerSku": {
```

Comment:
It'd be best if we can avoid any incorrect parameter values passed, so we should limit choices by providing the "allowedValues" (as above in "gatewayType"). I see that load balancers have "Basic" and "Standard" SKUs, so could you please add the "allowedValues" property with the two?